test/katello_test_helper.rb

```diff
@@ -171,6 +171,18 @@ def mock_active_records(*records)
       record.stubs(:reload).returns(record)
     end
   end
+
+  def equal_arrays(array1, array2)
+    array1.sort == array2.sort
+  end
+
+  def assert_equal_arrays(array1, array2)
+    assert equal_arrays(array1, array2)
```

Comment:
What about just doing:

```
assert_equal array1.sort, array2.sort
```

And refute equal for the other function without needing the `equal_arrays` function?